docs/CHANGELOG.asciidoc

```diff
@@ -37,6 +38,274 @@ This section summarizes the changes in each release.
+* Removes SLA percentage metric {pull}65718[#65718]
+* Removes dateline check for geo_shape queries, split geo_bounding_box queries that cross dateline into 2 boxes {pull}64598[#64598]
+* Handles case where fit to bounds does not match any documents {pull}66307[#66307]
+* Returns bounding box for static feature collection without joins {pull}66607[#66607]
+* Fixes legend icon color for custom categorial palettes {pull}67141[#67141]
+Metrics::
+* Disables search bar when live stream is on {pull}65491[#65491]
+* Ensures live stream always gets latest entries {pull}67935[#67935]
+* Fixes time picker layout issues on Inventory View {pull}66094[#66094]
+* Fixes validation for threshold values {pull}66281[#66281]
+* Prevents component errors from breaking the whole UI {pull}65456[#65456]
+* Fixes mistake in container ip field name {pull}66198[#66198]
+* Fixes time state bug {pull}67630[#67630]
+* Require filterQuery to be ES JSON {pull}64937[#64937]
+* Debounce onChange in alert creation UI {pull}65167[#65167]
+* Fixes default metric alert interval for new conditions {pull}66610[#66610]
+Platform::
+* Fixes inconsistent search behaviour in Advanced Settings {pull}64510[#64510]
+* Refactors nav links and header components {pull}66685[#66685]
+* SavedObjects bulkCreate API should return migrationVersion and strip the type &amp; namespace from the id {pull}65150[#65150]
+* Fixes task manager cancel warning {pull}63756[#63756]
+* New Kibana app link order {pull}67991[#67991]
+* Warning and link to support matrix for IE11 {pull}66512[#66512]
+Security::
+* Fix authentication loop when upgrading Kibana from 6.8 to 7.7+ {pull}67687[#67687]
+* Previously, users couldn't log in with SAML Identity Provider Initiated flow (e.g. from Okta Dashboard) if they already had 
+an existing, but expired session. Now it should be possible {pull}59686[#59686]
+* Fixes EncryptedSavedObjectsClientWrapper fails on stripEncryptedAttributes for saved object with errors {pull}61385[#61385]
+* Adds username/password validation to login form {pull}60681[#60681]
+SIEM::
+* Fixes import bug with non existent signals index {pull}65595[#65595]
+* Restricts ML rule modification to ML Admins {pull}65583[#65583]
+* Fixes timeline buildGlobalQuery {pull}68320[#68320]
+Security::
+* Fixes EncryptedSavedObjectsClientWrapper fails on stripEncryptedAttributes for saved object with errors {pull}61385[#61385]
+Uptime::
+* Refreshes index and also show more info to user regarding index/data {pull}62606[#62606]
+* Updates uptime ml job id to limit to 64 char {pull}64394[#64394]
+* Enables deselection of stale filters {pull}65523[#65523]
+* Unmounts uptime app properly {pull}66950[#66950]
+
+[float]
+[[deprecation-7.8.0]]
+=== Deprecation
+Security::
+* Deprecates {kib} user in favor of kibana_system user {pull}63186[#63186]
+
+[float]
+[[breaking-changes-7.8.0]]
+=== Breaking Changes
+Alerting::
+* [Fixes #63130] Changed actions API endpoints urls to follow Kibana STYLEGUIDE {pull}65936[#65936]
```

Comment:
I checked with the author of this PR and this is should have been labeled release_note:dev_docs.  So should be removed from here.

Also please add #62415 to the list of breaking changes.  It's a user-facing breaking change that is labeled as release_note:dev_docs.